ports/stm32/boards/NUCLEO_H563ZI/stm32h5xx_hal_conf.h

```diff
@@ -0,0 +1,19 @@
+/* This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2023 Damien P. George
+ */
+#ifndef MICROPY_INCLUDED_STM32H5XX_HAL_CONF_H
+#define MICROPY_INCLUDED_STM32H5XX_HAL_CONF_H
+
+// Oscillator values in Hz
+#define HSE_VALUE (8000000)
+#define LSE_VALUE (32768)
+#define EXTERNAL_CLOCK_VALUE (12288000)
+
+#include "boards/stm32h5xx_hal_conf_base.h"
```

Comment:
Please move this include down, after the STARTUP_TIMEOUT defines (see 82b4ab757603c52420f5e11470d0404ba93286ed).

Reply:
Done